### DISCUSSION TITLE
Does the scikit-fem support the sets of materials?

### DISCUSSION THREAD

**Discussion by SkivHisink** [kinnala/scikit-fem#702] [Q&A]
I have 3D objects with properties Pore ​​pressure, Poisson's ratio, Azimuth of maximum voltage, Density, Maximum and minimum voltage factor and Young's modulus in each cell. I need to find Stress and strain tensor in elements by solving elastic linear problem by the finite element method. Does is it possible to do for  solving the problem of linear elasticity of a finite element assembly?

**Comment by gdmcbain** [comment]
Yes. 

There's a simple example of properties varying by subdomain in [ex17 insulated wire](https://scikit-fem.readthedocs.io/en/latest/listofexamples.html#example-17-insulated-wire). The properties do vary by cell, or even by quadrature-point, but in this example are defined according to which cell the quadrature point belongs to, and which subdomain the cell beöongs to.

**Comment by kinnala** [comment]
I think our predefined linear elasticity operator given in `skfem.models.elasticity` does not support multiple materials (see #315) but that should be a simple enough modification.
